build.gradle

```diff
@@ -4,6 +4,9 @@ buildscript {
         maven {
             url 'https://plugins.gradle.org/m2/'
         }
+        maven {
+            url  "http://palantir.bintray.com/releases"
```

Comment:
I was actually having trouble with running this locally without adding bintray, I'll try to remove it and see if it isn't.